lib/wizard.js

```diff
@@ -0,0 +1,119 @@
+var _s = require('underscore.string');
+var fs = require('fs');
+var Q = require('q');
+var prompt = require('prompt');
+var merge = require('merge');
+module.exports = {
+  init: function (options, shouldDoOverrides) {
+    options.y = !!shouldDoOverrides;
+    prompt.message = '>>';
+    prompt.delimiter = ':'.green;
+    console.log("This utility will walk you through creating your very own presentation file.");
+    console.log("Press ^C at any time to quit.");
+    getPrompt(options).then(checkForFile).then(writeToFile).catch(console.log);
+  }
+};
+
+function getPrompt(options) {
+  var deferred = Q.defer();
+  var probableTitle = options.title || _s.capitalize(process.cwd().split('/').slice(-1)[0]);
+  var titlePrompt = [{
+      name: 'title',
+      description: 'Presentation title',
+      type: 'string',
+      default: probableTitle,
+      required: true
+    }];
+
+  prompt.start();
+  // skip the stuff!
+  if (options.y) {
+    prompt.override = {
+      title: probableTitle,
+      input: _s.slugify(probableTitle) + '.md',
+      output: 'index.html',
+      style: '   '
+    };
+  }
+  prompt.get(titlePrompt, function (err, res) {
+    var title = res.title;
+    if (err) { return deferred.reject(err); }
+    var promptSeries = [
+      {
+        name: 'input',
+        description: 'Input markdown file',
```

Comment:
Let's just go ahead and generate this for them automatically, based on the title. If they want to move it they can.